python_scripts/04_parameter_tuning_search.py

```diff
@@ -53,11 +53,12 @@
 
 # %%
 from sklearn.compose import ColumnTransformer
+from sklearn.compose import make_column_selector as selector
+
 from sklearn.preprocessing import OrdinalEncoder
 
-categorical_columns = [
-    'workclass', 'education', 'marital-status', 'occupation',
-    'relationship', 'race', 'native-country', 'sex']
+categorical_columns_selector = selector(dtype_include=object)
+categorical_columns = categorical_columns_selector(data)
```

Comment:
Maybe we can split using a new cell here just to show the output of using the selector?

Reply:
I guess we do it already in some notebooks previously e.g. here: 
https://inria.github.io/scikit-learn-mooc/python_scripts/03_categorical_pipeline.html#working-with-categorical-variables
![image](https://user-images.githubusercontent.com/1680079/99403694-0ae2e680-28eb-11eb-8e5a-35884bd6bc6e.png)

I am wondering whether it is better to do in all notebooks or only in one of the beginning at the beginning.